modules/cluster-logging-collector-fluentd.adoc

```diff
@@ -2,13 +2,11 @@
 //
 // * logging/cluster-logging-external.adoc
 
-[id="cluster-logging-collector-external_{context}"]
-= Configuring the Fluentd out_forward plug-in to send logs to an external log aggregator
+[id="cluster-logging-collector-fluentd_{context}"]
+= Configuring the Fluentd forward plug-ins to send logs to an external log aggregator
 
-You can configure Fluentd to send a copy of its logs to an external log
-aggregator instead of the default Elasticsearch instance using the Fluentd *forward*
-plug-in. From there, you can further process log records after the locally
-hosted Fluentd has processed them. 
+You can use the Fluentd *forward* plug-ins to send a copy of your logs to an external syslog server, 
+not the default Elasticsearch.
```

Comment:
"not the default Elasticsearch.", that is not correct. we sent logs to both.

Reply:
See: https://github.com/openshift/openshift-docs/pull/19650/files#diff-99ac0573c4348452e72233887a154360R11-R14
_You can configure {product-title} to send logs to both the internal Elasticsearch instance and an outside log aggregator using the '@type copy' plug-in within the Fluentd `output-application` configuration file. Specific instructions for this configuration are beyond the scope of this documentation._